source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,61 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``collection.findOne()`` method. 
```

Comment:
Thanks for the note. After rereading, I agree that it makes sense to immediately describe the functionality of the method in a usage example rather than starting from the signature. This is addressed in: https://github.com/mongodb/docs-golang/pull/2/commits/70cacf45b53ca05072ac20cec62b90df79be7642